templates/base.html

```diff
@@ -12,8 +12,6 @@
         {# Docs.rs styles #}
         <link rel="stylesheet" href="/-/static/vendored.css?{{ docsrs_version() | slugify }}" type="text/css" media="all" />
         <link rel="stylesheet" href="/normalize-{{ rustc_resource_suffix() }}.css" type="text/css" media="all" />
```

Comment:
Can you try to remove this file too?

Reply:
I'm pretty sure we already vendor it, there have been a lot of bugs related to it.
https://github.com/rust-lang/docs.rs/issues/992
https://github.com/rust-lang/docs.rs/issues/1005
https://github.com/rust-lang/docs.rs/pull/1059

Reply:
Yes!